pyramid_swagger/ingest.py

```diff
@@ -154,6 +156,33 @@ def get_swagger_schema(settings):
     return compile_swagger_schema(schema_dir, resource_listing)
 
 
+def get_swagger_spec(settings):
+    """Return a :class:`bravado_core.spec.Spec` constructed from
+    the swagger specs in `pyramid_swagger.schema_directory`. If
+    `pyramid_swagger.enable_swagger_spec_validation` is enabled the schema
+    will be validated before returning it.
+
+    :param settings: a pyramid registry settings with configuration for
+        building a swagger schema
+    :type settings: dict
+    :rtype: :class:`bravado_core.spec.Spec`
+    """
+    schema_dir = settings.get('pyramid_swagger.schema_directory', 'api_docs/')
+    with open(os.path.join(schema_dir, 'swagger.json'), 'r') as f:
+        spec_dict = simplejson.loads(f.read())
```

Comment:
Note: I am planning a later PR to check for an existance for a `swagger.json`. If it is not present combine individual swagger specs present in the directory, similar to something like this: http://pastebin.com/TzKAtKEA